Shared/Samples/Download vector tiles to local cache/DownloadVectorTilesToLocalCacheView.swift

```diff
@@ -33,10 +33,10 @@ struct DownloadVectorTilesToLocalCacheView: View {
     
     var body: some View {
         GeometryReader { geometry in
-            MapViewReader { mapView in
+            MapViewReader { mapViewProxy in
                 MapView(map: model.map)
                     .interactionModes(isDownloading ? [] : [.pan, .zoom])
-                    .onScaleChanged { model.maxScale = $0 * 0.1 }
+                    .onScaleChanged { model.updateMaxScale($0) }
```

Comment:
There is a problem within Swift compiler that doesn't allow me to do the shorthand 🤷 

https://forums.swift.org/t/converting-function-value-of-type-mainactor-to-void-loses-global-actor-mainactor/59155

```swift
.onScaleChanged(model.updateMaxScale)
```